gokart/task.py

```diff
@@ -359,7 +359,8 @@ def dump(self, obj: Any, target: Union[None, str, TargetOnKart] = None) -> None:
         PandasTypeConfigMap().check(obj, task_namespace=self.task_namespace)
         if self.fail_on_empty_dump and isinstance(obj, pd.DataFrame):
             assert not obj.empty
-        self._get_output_target(target).dump(obj, lock_at_dump=self._lock_at_dump)
+
+        self._get_output_target(target).dump(obj, lock_at_dump=self._lock_at_dump, task_params=super().to_str_params(only_significant=True))
```

Comment:
[must]
https://luigi.readthedocs.io/en/stable/api/luigi.task.html#luigi.task.Task.to_str_params

please specify only public paparameters
```suggestion
        self._get_output_target(target).dump(obj, lock_at_dump=self._lock_at_dump, task_params=super().to_str_params(only_significant=True, only_public=True))
```